lib/predictive_load/active_record_collection_observation.rb

```diff
@@ -1,13 +1,37 @@
 module PredictiveLoad::ActiveRecordCollectionObservation
 
   def self.included(base)
-    ActiveRecord::Relation.send(:include, RelationObservation)
+    if ActiveRecord::VERSION::MAJOR >= 5
+      ActiveRecord::Relation.send(:include, Rails5RelationObservation)
+    else
+      ActiveRecord::Relation.send(:include, RelationObservation)
```

Comment:
Shouldn't need to send anymore with new-ish ruby. I think this can just be `ActiveRecord::Relation.include()`